Project/Project/Models/Favorite.cs

```diff
@@ -11,6 +11,8 @@ public class Favorite
     // TODO: Add User_id as foreign key
     public int User_id { get; set; }
     public string Movie_id { get; set; }
+    public string movie_title { get; set; }
+    public string movie_poster { get; set; }
```

Comment:
To follow the standard please change to capitazed example Movie_title, Movie_poster